app/assets/javascripts/govuk-component/tasklist.js

```diff
@@ -21,6 +21,9 @@
 
     var rememberShownStep = false;
     var taskListSize;
+    var sessionStoreLink = 'govuk-task-list-active-link';
+    var activeLinkClass = 'pub-c-task-list__link-item--active';
```

Comment:
We should avoid binding to presentational classes if possible and use `js-` namespaced classes if possible.

This ensures when any refactoring takes place, it's obvious where a class is used.

Reply:
Agreed, except this is changing something that is only presentational. I can't see how I'd add another js- class that's tied to the presentational --active class without having to double up on everything.